apps/server/src/modules/learnroom/uc/course-export.uc.ts

```diff
@@ -1,19 +1,26 @@
 import { AuthorizationContextBuilder } from '@modules/authorization';
 import { AuthorizableReferenceType, AuthorizationReferenceService } from '@modules/authorization/domain';
-import { Injectable } from '@nestjs/common';
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
 import { Permission } from '@shared/domain/interface';
 import { EntityId } from '@shared/domain/types';
-import { CommonCartridgeVersion } from '../common-cartridge';
+import { CommonCartridgeVersion } from '../../common-cartridge';
+import { LearnroomConfig } from '../learnroom.config';
 import { CommonCartridgeExportService } from '../service/common-cartridge-export.service';
 
 @Injectable()
 export class CourseExportUc {
 	constructor(
+		private readonly configService: ConfigService<LearnroomConfig, true>,
 		private readonly courseExportService: CommonCartridgeExportService,
 		private readonly authorizationService: AuthorizationReferenceService
 	) {}
 
 	async exportCourse(courseId: EntityId, userId: EntityId, version: CommonCartridgeVersion): Promise<Buffer> {
+		if (!this.configService.get<boolean>('FEATURE_COMMON_CARTRIDGE_COURSE_EXPORT_ENABLED')) {
```

Comment:
Can you move this code to the private method like checkIsFeatureEnabled or so?

Reply:
Done.